docs/Contribute-to-AlmaLinux-Build-System.md

```diff
@@ -0,0 +1,212 @@
+---
+title: 'AlmaLinux Build System'
+---
+
+###### last updated: 2024-09-20
+
+# Contribute to AlmaLinux Build System
+
+The AlmaLinux Build System is managed by the [SIG/Build System](/sigs/Build-System) SIG. 
+
+The project is open for community contributions. If you are interested in contributing to the Build System project, we recommend you start by checking the AlmaLinux Build System project [milestones](https://github.com/AlmaLinux/build-system/milestones) for ongoing tasks and following the guidelines below.
+
+When you are ready to create a pull request with your suggested changes, please follow the **[commit guidelines](#commit-guidelines)** to keep our commit messages consistent across all repositories around the AlmaLinux Build System. 
+
+## Contribute to the codebase
+
+To contribute to development we recommend deploying the AlmaLinux Build System [locally](https://github.com/AlmaLinux/build-system/wiki/Deploy-on-local-and-remote-machines) and following these guide steps for possible modifications and commit guidelines.
+
+### Help Wanted
+
+We are seeking contributors to help us with:
+* Python and JavaScript Developers are needed to improve UI/UX.
+* Add the ability to run test VMs in different clouds (AWS, Azure, etc.).
+* Add the ability to use external repositories for testing scenarios (e.g. LTP for kernel, CentOS tests for different packages, openQA, etc.).
+* Add OpenStack backend driver support.
+* Add Azure backend driver support.
+* Improve ability to delete a build.
+* Add Kubevirt backend support.
+
+#### A new service's config
+* Add template to `roles/dev_deploy/templates` for config of a service. The name of the file should be `<name_of_target_config_file>.j2`.
+* Add description of config to `roles/dev_deploy/defaults/main/configs.yml`.
+
+#### A new directory
+* Add description to `roles/dev_deploy/defaults/main/common.yml`.
+
+#### A new docker container
+* Add a new description of a container's docker image to `roles/dev_deploy/defaults/main/docker_images.yml`.
+* Add a new description of a docker container to `roles/dev_deploy/defaults/main/docker_containers.yml`.
+
+## Commit Guidelines
```

Comment:
Actually, I'm not sure we should add the full commit guidelines here. What if we keep it as a standalone page? What do you think?

Reply:
I think keeping them in the contributor's guide is the right move, since each part of the AlmaLinux project might require different things.

Reply:
I don't mind keeping them here, it's just that, even though the commit guidelines are important, I don't really believe they deserve to be half of the contribution guide. That's my reasoning behind, but no strong opinion.

Reply:
From the other side, folks can notice them easier that also leads to their commits being up to your expectations

Reply:
Good point @sboldyreva, I totally agree, let's just keep them here